lib/string_tools.rb

```diff
@@ -105,6 +105,18 @@ def sanitize(text, options = {})
     def clear_control_characters(string)
       string.tr("\u0000-\u001f", '')
     end
+
+    def strip_all_tags_and_entities(string)
+      Sanitize.fragment(string.gsub(/&#([0-9]|10|11|12|13);|&nbsp;|\xc2\xa0|\s/, ''))
```

Comment:
Can we use `gsub!` instead `gsub` ?
And maybe we should add doc to methods with examples?